pyccel/codegen/printing/ccode.py

```diff
@@ -431,6 +431,16 @@ def fill_NumpyArange(self, expr, lhs):
 
     # ============ Elements ============ #
 
+    def _print_PythonAbs(self, expr):
+        func = "abs"
+        if expr.arg.dtype is NativeReal():
+            self._additional_imports.add("math")
+            func = "fabs"
+        elif expr.arg.dtype is NativeComplex():
+            self._additional_imports.add("complex")
+            func = "cabs"
+        return "{}({})".format(func, self._print(expr.arg))
```

Comment:
As a matter of style and readability, it is good habit not to redefine a variable  unless strictly necessary.

Here you are changing the value of `func` if the dtype matches a certain condition, but it is clearer and more natural to use an `else` condition instead:

```python
        if expr.arg.dtype is NativeReal():
            self._additional_imports.add("math")
            func = "fabs"
        elif expr.arg.dtype is NativeComplex():
            self._additional_imports.add("complex")
            func = "cabs"
        else:
            func = "abs"
```